test/Ionide.ProjInfo.Tests/Tests.fs

```diff
@@ -849,8 +849,10 @@ let testFCSmapManyProj toolsPath workspaceLoader (workspaceFactory: ToolsPath ->
 
         loader.Notifications.Add (function
             | WorkspaceProjectState.Loaded (po, knownProjects, _) -> pos <- Map.add po.ProjectFileName po pos)
-
-        let fcsPo = FCS.mapToFSharpProjectOptions parsed.Head parsed
+        let parsedC1 = parsed |> Seq.find(fun x ->
+            x.ProjectFileName.EndsWith(``sample3 Netsdk projs``.ProjectFile)
+        )
+        let fcsPo = FCS.mapToFSharpProjectOptions parsedC1 parsed
```

Comment:
Can't guarantee order so have to find our console app

Reply:
that makes sense 👍